CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.6.1
+
+- Optimize provider resource creation during prefetch to reduce system load
```

Comment:
Is that a fix?

Reply:
Yeah, it's a fix. Performance should not be impacted by prefetch (bug).